scripts/inference-dev

```diff
@@ -28,13 +28,17 @@ distDir="$cfiDir"/dist
 
 jdkPaths="${cfDir}"/jdk/annotated
 
-CFBuildDirs="${langtoolsDir}"/build/classes:"${annoToolsDir}"/scene-lib/bin:"${cfDir}"/dataflow/build:"${cfDir}"/javacutil/build:"${cfDir}"/stubparser/build:"${cfDir}"/framework/build:"${cfDir}"/checker/build
-CFIBuildDirs="${cfiDir}/bin"
+javacBuild="${langtoolsDir}"/build/classes
```

Comment:
Why split this variable out? CF always needs javac.

Reply:
Just for a better naming.. I.e. make `javacBuild` an explicit dependency of CF. If CF always needs javac, then it seems put them together would be more concise.

Reply:
CF always needs javac, so splitting this out probably doesn't add a lot.

Reply:
Ok, will put them back together.